src/main/java/com/google/api/codegen/discovery/ViewModelProvider.java

```diff
@@ -26,6 +26,7 @@
 import com.google.api.codegen.rendering.CommonSnippetSetRunner;
```

Comment:
Note that all the changes in this file are purely cosmetic, I thought it was way too verbose